coveralls.json

```diff
@@ -1,6 +1,8 @@
 {
   "skip_files": [
+    "lib/mix",
     "lib/alice.ex",
+    "lib/alice/console.ex",
```

Comment:
Why skip mix tasks and the console module?

Reply:
Honestly I was trying to find best practices for how to test a startup module like that, and didn't find answers I loved, so I checked how were were testing Alice's startup and found we were skipping it. That's when I just added it to the skip.

If you'd like to do differently I'm open to it. The first tests I wrote for it were just ugly, and I didn't find they added much value (did the application variables actually get set?), but maybe I was thinking about it wrong.

Reply:
More specifically to the mix task question, I suppose that's a holdover from my ruby tendencies where we didn't really test rake tasks. I definitely may have made that exclusion too broad though.

Reply:
I tested the mix task in alice_new and I think it turned out pretty nice

Reply:
You wouldn't need most of the helper functions like the temp dir etc since you aren't creating files. Look (here where it responds to io)[https://github.com/alice-bot/alice_new/blob/0fa9aeb2a26225a46af20daf3af028b5ab283e52/test/alice_new_handler_test.exs#L103]

Reply:
If you want to skip them for now, I'm going to write tests for those last 2 files soon and I can lump these in.

Reply:
It's good feedback and a good example, let me play with it for a bit and I'll get back to you.

Reply:
Ok, tests are up for both, and they've been removed from the skip_files list.